### PR TITLE
Add initial support for Patternfly in portal templates

### PR DESCRIPTION
**Pull request by jaredsmith**
This adds some very basic Patternfly support to the master portal template.

I could definitely use some help getting a better username displayed.

### CODE REVIEW

**Comment by jaredsmith** [comment]
Any idea why the packit/testing-farm-fedora-rawhide-x86_64 test hasn't completed yet?

**Comment by nullr0ute** [comment]
> Any idea why the packit/testing-farm-fedora-rawhide-x86_64 test hasn't completed yet?

Probably because there's a lack of resources.

File: tests/test_views_netboot.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def test_arch_file(self, *mocks):
 
         # Hacky fix to silence warning. If this causes failure, the internal
         # django test client API probably changed. This can be safely removed.
-        resp._closable_objects[0].close()
+        # resp._closable_objects[0].close()
```
Comment (puiterwijk) [inline tests/test_views_netboot.py]: Heh, so we both hit this! I fixed this in #38, so feel free to rebase on that after that's gotten merged.

**Comment by puiterwijk** [comment]
I saw a preview of this, and it looks awesome, thank you so much!
I'm hoping that after rebasing after #38 gets merged, the packit test will automatically get retriggered.
I am kind of wondering whether it didn't run because of the whitelist the Packit people maintain, so will reach out to them to see if that's what happened.

**Comment by puiterwijk** [comment]
\packit whitelist

**Comment by puiterwijk** [comment]
I filed https://github.com/packit-service/packit-service/issues/481 to ask for what happened here.

**Pull request merged**